Tavisca.Bootcamp.LanguageBasics.Exercise1/Equation.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1{
+class Equation{
+    public Equation(string operand1,string operand2,string operation,string result){
+        Operand1=operand1;
+        Operand2=operand2;
+        Operation=operation;
+        Result=result;
+    }        
+    public string Operand1 { get; private set; }
+    public string Operand2 { get; private set; }
+    public string Operation { get; private set; }
+    public string Result { get; private set; }
+    private bool IsValidValue(string actualValue,string expectedValue){
+        if(string.IsNullOrEmpty(actualValue)||string.IsNullOrEmpty(expectedValue))
```

Comment:
Invert this Condition.
You should always write valid condition check, if condition is satisfied  then do further operations or else just return default;
Like
if(string.IsNullOrEmpty(actualValue) == false && string.IsNullOrEmpty(expectedValue) == false)
 if(expectedValue.Length==actualValue.Length)
            return true;
return false;